### PR TITLE
feat: migrate blockquote component

### PR DESCRIPTION
**Pull request by araujogui**
Migrate `blockQuote` component and create a multiple paragraph test and story.
Issue #5242

### CODE REVIEW

File: components/Article/BlockQuote/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import type { PropsWithChildren } from 'react';
+import styles from './index.module.scss';
+
+const BlockQuote = ({ children }: PropsWithChildren) => (
+  <div className={styles.blockQuote}>{children}</div>
```
Comment (nschonni) [inline components/Article/BlockQuote/index.tsx:5]: There is a semantic element for this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

File: components/Article/BlockQuote/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.blockQuote {
```
Comment (nschonni) [inline components/Article/BlockQuote/index.module.scss:1]: Do you still need the class, or is the specificity enough if you just style the plain element?

Comment (ovflowd) [reply components/Article/BlockQuote/index.module.scss:1]: I suppose as a CSS module just the element name should be enough as the styles are scoped, but might be worth a test.

File: components/Article/BlockQuote/index.tsx (hunk 1 above)
Comment (araujogui) [reply components/Article/BlockQuote/index.tsx:5]: Just realized, when using the `blockquote` semantic element, the component styles conflicts with the ones in [styles/_base.scss#L89](https://github.com/nodejs/nodejs.org/blob/major/website-redesign/styles/_base.scss#L89)

**Comment by ovflowd** [comment]
@araujogui any update here?

File: components/Article/BlockQuote/index.tsx (hunk 1 above)
Comment (ovflowd) [reply components/Article/BlockQuote/index.tsx:5]: Feel free to go back to using `div`

File: components/Article/BlockQuote/index.module.scss (hunk 2 above)
Comment (manishprivet) [reply components/Article/BlockQuote/index.module.scss:1]: I believe CSS modules don't allow top level pure elements

**Comment by ovflowd** [comment]
@araujogui can you rebase your branch also?

**Comment by ovflowd** [comment]
@araujogui looks like your branch diverged from the base branch. I think it's easier to recheckout the base branch from upstream, then create your branch again (same name) and cherry pick the commits 🙃

**Comment by ovflowd** [comment]
Now it looks good! Great!

File: .eslintignore [hunk 3]
```diff
@@ -7,6 +7,9 @@ public/en/user-survey-report
 public/static/documents
 public/static/legacy
```
Comment (ovflowd) [inline .eslintignore]: This seems to be leftover of your rebase?

File: .eslintrc [hunk 4]
```diff
@@ -7,7 +7,9 @@
       "env": { "node": true, "es6": true }
     },
     {
-      "files": ["**/**.test.{js,jsx,ts,tsx}"],
+      "files": [
```
Comment (ovflowd) [inline .eslintrc]: Leftovers of rebase?

**Review by ovflowd — APPROVED** [review]
LGTM!

**Comment by ovflowd** [comment]
Hey, @araujogui can you rebase the branch (basically just accept the `package-lock.json` file from `major/website-redesign` and then simply run `npm install` so that it updates your lock file?)

**Comment by ovflowd** [comment]
@araujogui I recreated your PR branch for you, as it was polluted.

**Comment by ovflowd** [comment]
@nodejs/website can we get some reviews/approvals here?

**Review by mikeesto — APPROVED** [review]
Thanks for your first contribution

**Pull request merged**